tests/conftest.py

```diff
@@ -2005,7 +2005,7 @@ def rgw_deployments(request):
         pytest.skip("There is no RGW deployment available for this test.")
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
dont like the autouse of the fixture, could lead to problems in mcg-only deployment

Reply:
Also be aware that the autouse fixture is scheduled even in deployment itself which leads to the situation that we don't have any OCP/ODF installed yet. In the case something will fail here it will fail all the test cases as well.

Reply:
Thanks Amit, Petr - I removed the autouse